tests/testAutopas/tests/autopasInterface/Newton3OnOffTest.cpp

```diff
@@ -78,17 +77,18 @@ INSTANTIATE_TEST_SUITE_P(
             }()),
             ValuesIn([]() {
               std::vector<std::string> ret;
-              std::transform(
-                  autopas::allDataLayoutOptions.begin(), autopas::allDataLayoutOptions.end(), std::back_inserter(ret),
-                  [](autopas::DataLayoutOption d) -> std::string { return autopas::utils::StringUtils::to_string(d); });
+              auto allDataLayoutOptions = autopas::DataLayoutOption::getAllOptions();
```

Comment:
here, you should use the actual options for the parametrization and not the string representations of them.